packages/patternfly-3/patternfly-react-extensions/src/components/ClassificationBanner/ClassificationBanner.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * ClassificationBanner Component for PatternFly React
+ */
+class ClassificationBanner extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      closed: props.closed || false
+    };
+  }
```

Comment:
I question if the component should really keep the state rather than allow the application to maintain state by providing a callback from the close button. I do think at a minimum we should provide the callback to allow the application to handle the close and provide any necessary interactions (like a confirm or whatever).

Reply:
@jeff-phillips-18 I believe that I have added a callback now.